docs/en_US/SQuAD_evolution_examples.md

```diff
@@ -1,31 +1,17 @@
 # Automatic Model Architecture Search for Reading Comprehension
-This example shows us how to use Genetic Algorithm to find good model architectures for Reading Comprehension.
-
-## 1. Search Space
-Since attention and recurrent neural network (RNN) have been proven effective in Reading Comprehension.
-We conclude the search space as follow:
-
-1. IDENTITY (Effectively means keep training).
-2. INSERT-RNN-LAYER (Inserts a LSTM. Comparing the performance of GRU and LSTM in our experiment, we decided to use LSTM here.)
-3. REMOVE-RNN-LAYER
-4. INSERT-ATTENTION-LAYER(Inserts an attention layer.)
-5. REMOVE-ATTENTION-LAYER
-6. ADD-SKIP (Identity between random layers).
-7. REMOVE-SKIP (Removes random skip).
-
-![](../../examples/trials/ga_squad/ga_squad.png)
 
-### New version
-Also we have another version which time cost is less and performance is better. We will release soon.
+## Overview
+This example shows us how to use Genetic Algorithm to find good model architectures for Reading Comprehension.
 
-## 2. How to run this example in local?
+## Experimental
```

Comment:
NNI use "Experiment" instead of "Experimental" in most of the docs. Please confirm with others if you are on purpose to refactor this wording. Otherwise, please use the common used term.